diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8d0cc1bc..48c78998 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,13 +72,13 @@ endif()
 
 # Required libraries.
 if(USE_SDL2)
-  find_package(SDL2 2.28.2 REQUIRED)
+  find_package(SDL2 REQUIRED)
 endif()
 if(NOT WIN32 AND NOT ANDROID)
   find_package(CURL REQUIRED)
 endif()
 if(BUILD_QT_FRONTEND)
-  find_package(Qt6 6.5.1 COMPONENTS Core Gui Widgets Network LinguistTools REQUIRED)
+  find_package(Qt6 COMPONENTS Core Gui Widgets Network LinguistTools REQUIRED)
 endif()
 
 
