diff --git a/CMakeLists.txt b/CMakeLists.txt
index d35c2952..90d28ab6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -67,7 +67,7 @@ endif()
 
 # Required libraries.
 if(USE_SDL2)
-  find_package(SDL2 2.28.2 REQUIRED)
+  find_package(SDL2 REQUIRED)
 endif()
 if(NOT WIN32 AND NOT ANDROID)
   find_package(CURL REQUIRED)
@@ -77,7 +77,7 @@ if(NOT WIN32 AND NOT ANDROID)
   endif()
 endif()
 if(BUILD_QT_FRONTEND)
-  find_package(Qt6 6.5.1 COMPONENTS Core Gui Widgets Network LinguistTools REQUIRED)
+  find_package(Qt6 COMPONENTS Core Gui Widgets Network LinguistTools REQUIRED)
 endif()
 
 
